### DISCUSSION TITLE
How do you use Database Connection when self-hosting Teable?

### DISCUSSION THREAD

**Discussion by jaherron** [teableio/teable#459] [Q&A] [answered]
I see a feature in Teable's UI called Database Connection, but when I try to use it on a self-hosted instance it gives the error "PUBLIC_DATABASE_ADDRESS not found in env." I wonder what I should do (maybe add a line to the .env file in the compose directory I'm using) in order to solve the error.

I put this in the Discussions under Q&A, but I don't know if this should be opened as an issue or not. 

I used the compose files from /dockers/examples/standalone to make the containers.

**Comment by tea-artist** [comment]
We are aware of the problem and are working on it

**Comment by tea-artist** [comment, marked as answer]
Fixed! 
Download the latest example from /dockers/examples/standalone.
Go to `.env` file and change `127.0.0.1` in `PUBLIC_DATABASE_PROXY` to your external IP or domain name.
